### ISSUE TITLE
Doc: adding ueberauth to the applications list is no longer necessary

### ISSUE THREAD

**Issue by FractalWire** [ueberauth/ueberauth#117]
The doc mention both in the main repo and in the various strategy repos that you should add Ueberauth and its strategies to the applications list like so:

```elixir
def application do
  # Add the application to your list of applications.
  # This will ensure that it will be included in a release.
  [applications: [:logger, :ueberauth]]
end
```

For recent version of Elixir, this is no longer necessary and will, in fact, cause an error difficult to understand.

The doc should be updated to mention this change.

https://elixirforum.com/t/error-dbconnection-when-running-mix-phx-server/22652/2
https://www.amberbit.com/blog/2017/9/22/elixir-applications-vs-extra_applications-guide/

**Comment by doomspork** [comment]
Hi @FractalWire would you mind submitting a PR?

**Comment by FractalWire** [comment]
Sure ;)
https://github.com/ueberauth/ueberauth/pull/118

**Comment by Hanspagh** [comment]
Closed by #118
